VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json

```diff
@@ -942,7 +942,7 @@
     "you-may-use": "Gallwch ddefnyddio:",
     "laptop-or-desktop": "gliniadur neu gyfrifiadur bwrdd gwaith sydd â chamera a microffon",
     "apple-ipad": "iPad Apple (Safari yn unig)",
-    "if-you-cannot-access": "Os nad oes gennych ddyfais addas, ffoniwch y tîm gwrandawiadau fideo ar 0300 303 0655."
+    "if-you-cannot-access": "Os nad oes gennych ddyfais addas, ffoniwch y tîm gwrandawiadau fideo ar 0300 790 6234."
```

Comment:
Why is this being changed to the Scottish number? This might need to be updated to contain both but not as part of this ticket.

Reply:
@FarronF  For scottish translation, we need to use 0300 790 6234. The phone number in en.json remains unchanged

Reply:
This is Welsh not Scottish.

Reply:
@FarronF Updated phone number

Reply:
No we only support English and Welsh. It won't necessarily be based on country either.

Reply:
@FarronF I see. That's cool